app/code/Magento/Wishlist/Test/Unit/Controller/Index/UpdateTest.php

```diff
@@ -27,124 +30,124 @@
 class UpdateTest extends TestCase
 {
     /**
-     * @var Validator $formKeyValidator
+     * Wishlist item id
+     *
+     * @var int
+     */
+    private const ITEM_ID = 1;
```

Comment:
Prefix with `STUB_` as it means that is just "stupid replacement" for actual value, that does not matter for tests.